### ISSUE TITLE
Feature request: astroquery splatalogue filter to show astronomically observed transitions

### ISSUE THREAD

**Issue by travisthieme** [astropy/astroquery#1599]
Hi there,

Just wanted to see if it’s possible to add an option to the Splatalogue query to only show astronomically observed transitions. Since there’s already the option to only show NRAO recommended transitions, maybe it’s simple to implement.

They have this search filter on their website (circled in red in picture below). 

<img width="1681" alt="Screen Shot 2019-12-10 at 01 04 35" src="https://user-images.githubusercontent.com/40456866/70453991-0e6fca00-1aed-11ea-92eb-755dd03261f4.png">

Not sure if it’s in the future plan already, so I thought I can at least ask! Thank you!

**Comment by keflavich** [comment]
I believe this is already the default, but the way it is specified is not terribly obvious from the documentation.
```
exclude : list
    Types of lines to exclude.  Default is:
    (``'potential'``, ``'atmospheric'``, ``'probable'``)
    Can also exclude ``'known'``.
    To exclude nothing, use 'none', not the python object None, since
    the latter is meant to indicate 'leave as default'
```
so, in other words, the default is to do something like `Splatalogue.query_lines(..., exclude=['potential', 'atmospheric', 'probable'])`.  Could you verify whether that achieves the same results as the web form checkbox?  If not, we should investigate further.

**Comment by travisthieme** [comment]
When I use the default setting, it gives me over 100 lines instead of 3. I added a picture of part of the list after my code example.

**A working example of my code would be:**
from astroquery.splatalogue import Splatalogue,utils
from astropy import units
line_query = Splatalogue.query_lines(219.92\*units.GHz,219.98\*units.GHz,top20='ism_darkcloud',only_NRAO_recommended=True)
print ('Number of molecules found: {}'.format(len(line_query)))
minitable = utils.minimize_table(line_query)
minitable.pprint(max_lines=100,max_width=200)

<img width="1011" alt="Screen Shot 2019-12-10 at 01 55 35" src="https://user-images.githubusercontent.com/40456866/70455768-34e33480-1af0-11ea-8bbc-5f4f9f19e841.png">

**Now, in the Splatalogue advanced search query, using the same criteria I get the same amount of results (118 molecules):**

<img width="1679" alt="Screen Shot 2019-12-10 at 02 00 46" src="https://user-images.githubusercontent.com/40456866/70456418-67d9f800-1af1-11ea-9f4f-a0b68422e598.png">

**Then, when I click the 'Show ONLY astronomically-observed transitions' check box, I get 3 molecules:**

<img width="1678" alt="Screen Shot 2019-12-10 at 02 02 56" src="https://user-images.githubusercontent.com/40456866/70456427-6c061580-1af1-11ea-80a2-55b9108af43d.png">

**Comment by travisthieme** [comment]
Also, for another reference I checked the boxes you mentioned for the 

exclude : list
    Types of lines to exclude.  Default is:
    (``'potential'``, ``'atmospheric'``, ``'probable'``)
    Can also exclude ``'known'``.
    To exclude nothing, use 'none', not the python object None, since
    the latter is meant to indicate 'leave as default'

and got 118 molecules again.

<img width="1680" alt="Screen Shot 2019-12-10 at 02 11 11" src="https://user-images.githubusercontent.com/40456866/70456987-95737100-1af2-11ea-88ea-13608260d2f8.png">

**Comment by keflavich** [comment]
Yep, you're right, this is a new option we don't support yet.  See #1600.
